### DISCUSSION TITLE
I created a GitHub account twice!!

### DISCUSSION THREAD

**Discussion by ghost** [community/community#141846] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Before the account I use now, I had created a GitHub account under the name Manuel Jaime. The problem is that I can't access it. And whenever I go to GitHub and search for my name, that account always appears.

hey, I didn't do any activity on that account and it's already old.

I would like the administrators to help me delete that account.

**Comment by madfcat** [comment, marked as answer]
You should try to contact the [support](https://docs.github.com/en/support/contacting-github-support).
There is a way to [create a ticket](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket) and ask your account to be deleted.

**Comment by ranjith-fullstack** [comment]
You should try to contact the [support](https://docs.github.com/en/support/contacting-github-support).
There is a way to [create a ticket](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket) and ask your account to be deleted.
use this steps it helps you
